src/components/SelectInput/SelectInput.stories.tsx

```diff
@@ -13,13 +13,19 @@ const meta: Meta<typeof SelectInput> = {
 
 export default meta;
 
+type Option = {
+  value: string;
+  label: string;
+};
+
+const options: Option[] = [
+  { value: 'chocolate', label: 'Chocolate' },
+  { value: 'strawberry', label: 'Strawberry' },
+  { value: 'vanilla', label: 'Vanilla' },
+];
+
 export const Default = () => {
   const [value, setValue] = useState(null);
-  const options = [
-    { value: 'chocolate', label: 'Chocolate' },
-    { value: 'strawberry', label: 'Strawberry' },
-    { value: 'vanilla', label: 'Vanilla' },
-  ];
```

Comment:
These are intentionally repeated in each story so that the code samples show a complete example. Please add these back into each story.

![Screenshot 2024-10-07 at 4 24 42 PM](https://github.com/user-attachments/assets/bd5a748f-a1ac-4394-b5a0-8872645344a2)